CONTRIBUTING.md

```diff
@@ -2,7 +2,7 @@
 
 ## Bug Reports
 
-Please [open an issue](https://github.com/serokell/ariadne/issues) if
+Please [open an issue](https://issues.serokell.io/issues/AD) if
```

Comment:
Can you make this link lead straight to 'new issue' page?

Reply:
If you mean https://issues.serokell.io/newIssue?project=AD, then the reason why I didn't do it is that you need to login, otherwise you'll see https://i.imgur.com/8umODS7.png and it may look scary.
I think I should mention here that one needs to login, though.

Reply:
Ok, I think I fixed it properly (decided to add `newIssue` link eventually).